src/main/java/mate/academy/internetshop/dao/impl/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -1,37 +1,40 @@
 package mate.academy.internetshop.dao.impl.jdbc;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import java.sql.Connection;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+
+import java.util.ArrayList;
+import java.util.List;
+
 import mate.academy.internetshop.dao.ItemDao;
 import mate.academy.internetshop.models.Item;
+import org.apache.log4j.Logger;
 
 public class ItemDaoJdbcImpl extends AbstractClass<Item> implements ItemDao {
-    private static String DB_NAME = "internet_shop";
+    private static final String DB_NAME = "internet_shop";
+    private static final Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
 
     public ItemDaoJdbcImpl(Connection connection) {
         super(connection);
     }
 
     @Override
     public Item add(Item item) {
-        String query = "INSERT INTO " + DB_NAME + ".items (name, price)"
-                + " values('" + item.getName() + "', " + item.getPrice() + ");";
+        String query = String.format("INSERT INTO %s.items (name, price) values('%s', %s);",
```

Comment:
change %s to %f  because `price` is double. Check it in all queries      
 String query = String.format("INSERT INTO %s.items (name, price) values('%s', %f);", ?